spec/controllers/comments_controller_spec.rb

```diff
@@ -52,5 +53,38 @@
       }
       expect(response.code).to eq("422")
     end
+    describe "POST #upvote" do
```

Comment:
Add a line break to make it more clear where one set of tests ends and the next begins.